src/underactuated/pyplot_visualizer.py

```diff
@@ -48,20 +50,56 @@ def __init__(self, draw_timestep=0.033333, facecolor=[1, 1, 1],
 
         self.ax.axis('equal')
         self.ax.axis('off')
+        self.record = False
```

Comment:
I'm pretty OK with the start_recording -> simulate -> get_recording pipeline (it's very clear in the top-level code what's going on), but it's contrary to how SignalLogger works -- the equivalent here would be having record be an optional kwarg in the constructor and offer a reset_recording method. (And maybe remove stop_recording, but it's not hurting anyone.) The SignalLogger interface feels a little more clean IMO in terms of being system block that is initialized correctly for its intended use exactly when it's constructed (instead of relying on its functionality being changed later on).